src/apiCalls.test.js

```diff
@@ -0,0 +1,430 @@
+
+        it('should return an array of farmers markets', () => {
+            expect(getMarketsByVendor()).resolves.toEqual(mockResponse);
+        })
+
+        it('should return an error for response that is not ok', () => {
+            window.fetch = jest.fn().mockImplementation(() => {
+                return Promise.resolve({
+                    ok: false
+                });
+            });
+            expect(getMarketsByVendor()).rejects.toEqual(Error('Error fetching markets'))
+        })
+})
+    
+    describe('deleteMarketVendorLink', () => {
+        let mockId = 1
+        let mockResponse = [{
+            id:2,
+            name: "Highland Farmers",
+            description: "1500 Boulder Street, Denver, Colorado, 80211"
+        },
+        {   id:1,
+            name: "Farmers",
+            address: "1500 hello Street, Denver, Colorado, 80211"
+        }
+        ]
+
+        const mutation = {
+            "query": `
+        mutation{deleteMarketVendor(id: ${mockId})}`
+        }
+
+        let mockOptions =  {
+            method: 'POST',
+            body: JSON.stringify(mutation),
+            headers: {
+              "Content-Type": "application/json",
+              "Accept": "application/json"
+            }
+          }
+    
+    
+      beforeEach(() => {
+        window.fetch = jest.fn().mockImplementation(() => {
+          return Promise.resolve({
+            ok: true,
+            json: () => Promise.resolve(mockResponse)
+          });
+        });
+      });
+    
+        it.skip('should be passed the correct url', () => {
+            deleteMarketVendorLink(mockId)
+            expect(window.fetch).toHaveBeenCalledWith(process.env.REACT_APP_BACKEND_URL + `/graphql`, mockOptions);
+            })
+    
+        it('should delete vendor markets', () => {
+            expect(deleteMarketVendorLink()).resolves.toEqual(mockResponse);
+            })
+    
+        it('should return an error for response that is not ok', () => {
+                window.fetch = jest.fn().mockImplementation(() => {
+                    return Promise.resolve({
+                        ok: false
+                    });
+                });
+                expect(deleteMarketVendorLink()).rejects.toEqual(Error('Error deleting'))
+            })
+        })
+    
+    describe('getVendorsByMarketId', () => {
+        let mockId = 1
+        let mockResponse = [{
+            name: "Highland Farmers",
+            description: "1500 Boulder Street, Denver, Colorado, 80211"
+        },
+        {
+            name: "Farmers",
+            address: "1500 hello Street, Denver, Colorado, 80211"
+        }
+        ]
+    
+    
+      beforeEach(() => {
+        window.fetch = jest.fn().mockImplementation(() => {
+          return Promise.resolve({
+            ok: true,
+            json: () => Promise.resolve(mockResponse)
+          });
+        });
+      });
+    
+        it.skip('should be passed the correct url', () => {
```

Comment:
Nice work on testing these. It is weird that these are not passing and form what we talked about and hear from some other groups it seems like this is due to how things are set up in the backend for taking in the url. We can check with them on it if we have the time tomorrow.